w3/models.py

```diff
@@ -9,15 +9,18 @@ class ProgrammeTypes(IntEnum):
 
 class Sectors(IntEnum):
     WASH = 0
+    PGI = 1
+    CEA = 2
+    MIGRATION = 3
+    HEALTH = 4
+    DRR = 5
+    SHELTER = 6
+    PREPAREDNESS = 7
 
 class Statuses(IntEnum):
-    IN_PROGRESS = 0
+    ONGOING = 0
```

Comment:
Sorry for the nitpicking @szabozoltan69 but can we change this to:

    PLANNED = 0
    ONGOING = 1
    COMPLETED = 2

Just seems like a more logical ordering - first some thing is planned, then ongoing, then finally completed. Let me know if you'd rather I make this change. These changes look good 👌

Reply:
Good Idea